src/app/cube/new-home/learning-object-info/learning-objects/learning-objects.component.html

```diff
@@ -1 +1,51 @@
-<p>learning-objects works!</p>
+<div class="wrapper">
+    <div class="learning-object-info">
+        <h1>Modular self-contained cyber curricula</h1>
+        <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris 
+            nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in 
+            reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
+            Excepteur sint occaecat cupidatat non proident, 
+            sunt in culpa qui officia deserunt mollit anim id est laborum.
+        </p>
```

Comment:
don't forget to update this

Reply:
Mike and I will work on getting copy for each of these sections and put in a PR once everything is merged